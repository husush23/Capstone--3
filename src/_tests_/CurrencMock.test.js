/*eslint-disable*/
import React from 'react';
import {render} from '@testing-library/react';
import CurrencMock from './mocks/CurrencyMock';

describe('currency component', () => {
  test('Renders Currency correctly', () => {
    render(<CurrencMock />);
  });
  test('renders Currency correctly', () => {
    const {asFragment} = render(<CurrencMock />);
    expect(asFragment()).toMatchSnapshot();
  });
});
