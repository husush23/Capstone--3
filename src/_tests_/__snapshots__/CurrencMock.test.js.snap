// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currency component renders Currency correctly 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <div
      class="card"
    >
      <h3>
        Pair name
      </h3>
      <div
        class="values"
      >
        <div
          class="buy"
        >
          <span>
            ↗
          </span>
            
          <span>
            34.000000
          </span>
        </div>
        <div
          class="sell"
        >
          <span>
            ↘
          </span>
            
          <span>
            35.000000
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
